core/templates.yml

```diff
@@ -326,6 +340,24 @@ paths:
     sequence_asset_publish_maya_scene:      '@sequence_asset_publish_area_scene/@sequence_asset_version_name.{maya.ext}'
     shot_asset_publish_maya_scene:          '@shot_asset_publish_area_scene/@shot_asset_version_name.{maya.ext}'
 
+    # ZBRUSH SCENES
+    project_publish_zbrush_scene:             '@project_publish_area_scene/@project_version_name.{zbrush.ext}'
+    sequence_publish_zbrush_scene:            '@sequence_publish_area_scene/@sequence_version_name.{zbrush.ext}'
+    shot_publish_zbrush_scene:                '@shot_publish_area_scene/@shot_version_name.{zbrush.ext}'
+
+    project_asset_publish_zbrush_scene:       '@project_asset_publish_area_scene/@project_asset_version_name.{zbrush.ext}'
+    sequence_asset_publish_zbrush_scene:      '@sequence_asset_publish_area_scene/@sequence_asset_version_name.{zbrush.ext}'
+    shot_asset_publish_zbrush_scene:          '@shot_asset_publish_area_scene/@shot_asset_version_name.{zbrush.ext}'
+
+    # SUBSTANCE PAINTER SCENES
+    project_publish_substance_painter_scene:  '@project_publish_area_scene/@project_version_name.{substance_painter.ext}'
+    sequence_publish_substance_painter_scene: '@sequence_publish_area_scene/@sequence_version_name.{substance_painter.ext}'
+    shot_publish_substance_painter_scene:     '@shot_publish_area_scene/@shot_version_name.{substance_painter.ext}'
+
+    project_asset_publish_substance_painter_scene:  '@project_asset_publish_area_scene/@project_asset_version_name.{substance_painter.ext}'
+    sequence_asset_publish_substance_painter_scene: '@sequence_asset_publish_area_scene/@sequence_asset_version_name.{substance_painter.ext}'
+    shot_asset_publish_substance_painter_scene:     '@shot_asset_publish_area_scene/@shot_asset_version_name.{substance_painter.ext}'
+
```

Comment:
We can leave this just as `substance_scene` instead of `substance_painter_scene` if we need to support other substance file formats we can add those later.

Reply:
ok
i will update